quickstart-node/src/index.ts

```diff
@@ -15,10 +16,9 @@ app.use(cookieParser());
 
 // Routes
 app.use('/auth', authRouter);
-
-app.get('/', async (req: Request, res: Response) => {
-  res.send('Express + TypeScript Server ');
-});
+app.use('/monday', mondayRouter);
+app.use('/queue', queueRouter);
+app.use('/mail', mailRouter);
 
```

Comment:
Maybe leave a basic "healthy check" route as the '/'

Reply:
Not related to this comment but to the comment below (I can't comment on that comment for some reason...):
shortLivedToken, like its name, it is short lived. So that is why I create the api client in each request and not save it as a global class object. I did exactly as the example you gave me. I create the api client using the token in each function. Do you think there is another better way to implement this?